.github/workflows/ci.yaml

```diff
@@ -65,9 +65,9 @@ jobs:
           path: packages
 
       - run: npm run test:ci
-      - run: npm exec -- coveralls < .cache/coverage/lcov.info
-        env:
-          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      - uses: coverallsapp/github-action@643bc377ffa44ace6394b2b5d0d3950076de9f63 # v2.3.0
```

Comment:
Points to `2.3.0` tag.

https://github.com/coverallsapp/github-action/commit/643bc377ffa44ace6394b2b5d0d3950076de9f63

![Screenshot 2024-09-27 at 11 57 39 AM](https://github.com/user-attachments/assets/3d2e965c-d4ae-47fa-bcb6-2ab75abcb950)